falkordb-node/node-entrypoint.sh

```diff
@@ -276,11 +276,11 @@ wait_until_node_host_resolves() {
 
 get_master() {
   master_info=$(redis-cli -h $SENTINEL_HOST -p $SENTINEL_PORT --user $FALKORDB_USER -a $FALKORDB_PASSWORD $TLS_CONNECTION_STRING --no-auth-warning SENTINEL get-master-addr-by-name $MASTER_NAME)
-  redisRetVal=$?
+
   echo "Master Info: $master_info"
 
   # If RUN_SENTINEL is 1 and could not connect to sentinel, wait and try again
-  if [[ $RUN_SENTINEL -eq 1 && $redisRetVal -ne 0 ]]; then
+  if [[ $RUN_SENTINEL -eq 1 && -z $master_info && ! $HOSTNAME =~ ^node.*0 ]]; then
```

Comment:
Can you explain why you added the hostname check please?

Reply:
if I do not add  $HOSTNAME =~ ^node.*0  it will go into a loop. This command : "SENTINEL get-master-addr-by-name $MASTER_NAME" returns an empty response with no error message when it does not find a master.
now consider node-sz/mz-0 which needs to be master. I will go into a loop since there is not master.
consider this:
if [[ -z $FALKORDB_MASTER_HOST ]]; then
    FALKORDB_MASTER_HOST=$NODE_HOST
    FALKORDB_MASTER_PORT_NUMBER=$NODE_PORT
    IS_REPLICA=0
    return
  fi